mage_ai/data_cleaner/transformer_actions/utils.py

```diff
@@ -47,10 +47,10 @@ def build_action_variables(
 
 def build_transformer_action(
     df: DataFrame,
-    action_type: Union[ActionType, str],
-    action_arguments: List[str] = [],
+    type: Union[ActionType, str],
```

Comment:
type is a keyword, let's try not using it as an argument name.

Reply:
Changed back to `action_type`